src/lib/CreateFormMixin.js

```diff
@@ -99,7 +100,11 @@ module.exports = {
     // Global errors
```

Comment:
Line 98 needs only concatenate `typeError` if it is an array; if it is true, it should not concatenate it. You could do this in a number of ways:

```
if (typeError.length) errors = errors.concat(typeError);
```
or 
```
if (typeError instanceof Array) errors = errors.concat(typeError);
```
or 
```
if (typeError !== true) errors = errors.concat(typeError);
```

Personally, I prefer checking `typeError.length`